src/router/index.js

```diff
@@ -1,7 +1,14 @@
 import { createRouter, createWebHistory } from "vue-router";
-import Record from "..//components/Record.vue";
+import Home from "@/views/Home.vue";
+import Record from "@/views/Record.vue";
 
 const routes = [
+  {
+    path: "/",
+    name: "Home",
+    component: Home,
```

Comment:
I think you can just do `component: {},` here and note even need a `src/views/Home.vue`

Reply:
(this is not a required change, just sharing info I had stumbled across the other day in case it felt better to use here.. I don't have strong feelings either way)